docs/swagger.yaml

```diff
@@ -493,7 +629,53 @@ components:
       schema:
         type: string
       description: Bearer <MSI token issued to the identity of the client>
+    Transaction-Type:
+      name: paymentType
+      in: path
+      required: true
+      schema:
+        type: string
+        enum:
+          - ecom
+          - recurring
+      description: 'The type of transaction {ecom, recurring} '
+    Order-Id:
+      name: orderId
+      in: path
+      required: true
+      schema:
+        type: string
+        pattern: '^[0-9A-Za-z-_\.]+$'
+      description: 'The orderId of the transaction, unique and managed by merchant'
+  responses:
+    400BadRequest:
+      description: Example response
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    409Conflict:
+      description: Example response
+      content:
+        application/json:
+          schema:
+            properties:
+              id:
+                type: string
+    404NotFound:
+      description: Example response
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+    201Created:
```

Comment:
This need an example to get rid of the "string" in the docs
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/47554984/161014626-080aea42-c2a3-4995-9e0d-18f125dc32cd.png">

Reply:
Maybe not that important though